src/com/owncloud/android/files/services/FileDownloader.java

```diff
@@ -536,7 +536,7 @@ private void saveDownloadedFile() {
         file.setFileLength((new File(mCurrentDownload.getSavePath()).length()));
         file.setRemoteId(mCurrentDownload.getFile().getRemoteId());
         mStorageManager.saveFile(file);
-        mStorageManager.triggerMediaScan(file.getStoragePath());
+        FileDataStorageManager.triggerMediaScan(file.getStoragePath());
```

Comment:
Instead of using triggerMediaScan as static method, I would delete the static label and keep it using as an instance method via mStorageManager.

Reply:
@theScrabi this is still pending

Reply:
I tried to make it non static, but `triggerMediaScan()` is used in `UploadFileOperation` which does not provide all the necessary classes for creating a whole `FileDataStorageManager`. Creating and initializing those clases there would not make sense, since only `triggerMediaScan()` gets called, and that does not need them.

Reply:
Can't you use `getStorageManager().triggerMediaScan()` in UploadFileOperation?